src/main.c

```diff
@@ -302,7 +286,24 @@ main (int argc, char *argv[])
                               time_now, time_str_appended, current_waste_num);
 
             if (info_status == 0)
+            {
```

Comment:
Clearly these edits shouldn't be here. ;) I merged that patch in a few days ago. I'd suggest having a look at this [contributing page](https://github.com/andy5995/rmw/wiki/Contributing) to see how to avoid this - and other conflicts - in the future. Please don't hesitate to let me know if you have any questions.